### PR TITLE
Auction Admin Form Datepickers use EST/EDT

### PR DESCRIPTION
**Pull request by harrisj**
The admin form should specify auction times in the Eastern timezone

### CODE REVIEW

**Comment by jessieay** [comment]
This looks AWESOME. 

Did you look any further into whether date updates are being persisted with our new datepickers?

File: config/locales/simple_form.en.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@ en:
       default_message: "Please review the problems below:"
     labels:
       auction:
-        started_at: "Start date and time (in UTC)"
-        ended_at: "End date and time (in UTC)"
-        delivery_due_at: "Deliery deadline date and time (in UTC)"
+        started_at: "Start date and time (in ET)"
```
Comment (jessieay) [inline config/locales/simple_form.en.yml]: gotta love I18n :)

File: features/admin_edits_auction.feature [hunk 2]
```diff
@@ -11,7 +11,7 @@ Feature: Admin edits auctions in the admins panel
     And I edit the new auction form
     And I click to create an auction
     Then I should see the auction's title
-    And I should see the auction's deadline
+    And I should see when bidding starts and ends in ET
```
Comment (jessieay) [inline features/admin_edits_auction.feature]: YES

Comment (jessieay) [inline features/admin_edits_auction.feature]: we already have a step for this? Amazing

**Comment by harrisj** [comment]
I believe so, but let me double-check

File: spec/models/date_time_parser_spec.rb [hunk 3]
```diff
@@ -2,22 +2,23 @@
 
 describe DateTimeParser do
   describe '#parse' do
-    it 'returns a datetime based on attrs passed in' do
+    it 'returns an EST datetime based on attrs passed in' do
       fake_date = "2016-05-26"
       fake_hour = "01"
       fake_minute = "15"
-      fake_meridien = "AM"
+      fake_meridian = "PM"
```
Comment (jessieay) [inline spec/models/date_time_parser_spec.rb]: 🔡

File: app/views/admin/auctions/_time_selector.html.erb [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <div>
-<select class="time-input" name="auction[<%=field%>_at(1i)]">
+<select class="time-input" name="auction[<%=field%>_at(1i)]" id="auction_<%=field%>_at_1i">
```
Comment (jessieay) [inline app/views/admin/auctions/_time_selector.html.erb]: ah! was it the ids that were causing them not to persist?

**Comment by jessieay** [comment]
@harrisj very tiny cc error here 👶

File: app/presenters/dc_time_presenter.rb [hunk 5]
```diff
@@ -1,8 +1,10 @@
 class DcTimePresenter < Struct.new(:time)
   FORMAT = "%B %d, %Y %r".freeze
+  TIME_ZONE_NAME = 'Eastern Time (US & Canada)'.freeze
```
Comment (jessieay) [inline app/presenters/dc_time_presenter.rb]: YAS

File: app/views/admin/auctions/_time_selector.html.erb (hunk 4 above)
Comment (harrisj) [inline app/views/admin/auctions/_time_selector.html.erb]: No, I just need the IDs for Capybara ;)

Comment (jessieay) [inline app/views/admin/auctions/_time_selector.html.erb]: oh ok cool. I think capybara can also use `name` field, but this is fine.

**Comment by jessieay** [comment]
This is a beautiful thing 🙏

File: app/views/admin/auctions/_time_selector.html.erb (hunk 4 above)
Comment (harrisj) [inline app/views/admin/auctions/_time_selector.html.erb]: That wasn't working for me for some reason. Dunno. Anyhow, I think the persistence issue was loading with no timezone + AM/PM and other time saving issues

File: config/locales/simple_form.en.yml (hunk 1 above)
Comment (harrisj) [inline config/locales/simple_form.en.yml]: I do! Thank you for this

**Pull request merged**